ext/smoke/packages/steps/setup-master.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+set -e
+set -x
```

Comment:
Should this be turned off before we promote?

Reply:
I think I prefer leaving it in; if it's gone, and e.g. an rpm command fails, it'd be helpful to know the exact command being executed to dive in, saving the need to repeat with the option enabled. A small amount of verbosity seems worth it, for now.

Reply:
After a bit more consideration, this should indeed be removed. My concern was silent failures, but I think that will be solved by the `curl -f` fix suggested by Branan.